### PR TITLE
 createdPost/Delete

### PR DESCRIPTION
**Pull request by hewonjeong**
- add createdPost/Delete

### CODE REVIEW

File: src/server/users/models.js [hunk 1]
```diff
@@ -220,6 +220,28 @@ export class CreatedPostManager {
       return cb(err);
     });
   }
+  static genKey(userKey, entityKey, state) {
```
Comment (ovekyc) [inline src/server/users/models.js:207]: How about move this function to Key utils?

**Comment by ovekyc** [comment]
LGTM

**Comment by festinalente91** [comment]
LGTM

**Comment by hewonjeong** [comment]
lgtm

**Pull request merged**